modules/users/client/config/users-admin.client.routes.js

```diff
@@ -28,6 +28,18 @@
           pageTitle: '{{ userResolve.displayName }}'
         }
       })
+      .state('admin.adduser', {
+        url: '/adduser',
+        templateUrl: '/modules/users/client/views/admin/adduser.client.view.html'//,
+        //controller: 'UserController',
```

Comment:
this looks pretty necessary. we definitely need a controller for the view and we shouldn't commit commented code